addOns/pscanrules/src/main/java/org/zaproxy/zap/extension/pscanrules/BigRedirectsScanRule.java

```diff
@@ -116,20 +158,16 @@ public String getName() {
         return Constant.messages.getString(MESSAGE_PREFIX + "name");
     }
 
-    private String getDescription() {
-        return Constant.messages.getString(MESSAGE_PREFIX + "desc");
-    }
-
     private String getSolution() {
         return Constant.messages.getString(MESSAGE_PREFIX + "soln");
     }
 
-    private String getReference() {
-        return Constant.messages.getString(MESSAGE_PREFIX + "refs");
-    }
-
     @Override
     public Map<String, String> getAlertTags() {
         return ALERT_TAGS;
     }
+
+    public String getHelpLink() {
+        return "https://www.zaproxy.org/docs/desktop/addons/passive-scan-rules/#big-redirect-detected-potential-sensitive-information-leak";
```

Comment:
Use the ID of the scan rule instead.

Reply:
You mean the same link as added to the help file?

Reply:
I had followed what was done and merged in: https://github.com/zaproxy/zap-extensions/pull/5094/

Reply:
The ID is less likely to change than the name. https://github.com/zaproxy/zap-extensions/pull/5096#discussion_r1420727027

Reply:
Okay, thanks